docs/semgrep-ci/semgrep-ci-with-app.md

```diff
@@ -0,0 +1,332 @@
+---
+slug: semgrep-ci-with-app
+append_help_link: true
+description: ""
+title: Running Semgrep in CI with Semgrep App
+hide_title: true
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+import CiScheduling from "/src/components/CiScheduling.mdx"
+import CiIgnoringFiles from "/src/components/CiIgnoringFiles.mdx"
+import DiffAwareScanning from "/src/components/DiffAwareScanning.mdx"
+
+# Running Semgrep in continuous integration (CI) with Semgrep App
+
+Run Semgrep in your continous integration (CI) pipeline to scan your repository for code vulnerabilities and other issues. Connect your CI pipeline with Semgrep App to:
+
+* Block pull or merge requests (PRs or MRs) based on the rule that generated the finding.
+* Scan many repositories and manage their findings in bulk.
+* Triage (ignore) false-positive findings from low-performing rules.
```

Comment:
In my mind ignoring is one way to triage but not the only way to triage. Similarly, ignoring false positives is just one way to deal with them (another might be to tweak the rule, for example). I don’t have a specific copy edit recommendation because I’m not quite sure what the intent is here.

One suggestion unrelated to the above may be to change “low-performing” to “noisy.”

Reply:
Changed to "Ignore false-positive findings from noisy rules." The idea is to have specific features that App has that stand-alone CI jobs don't.